### ISSUE TITLE
compatibility with livewire

### ISSUE THREAD

**Issue by anurat** [BKWLD/laravel-pug#100]
Hello,

Would it be possible to use Laravel pug for pug template together with Laravel livewire?
A quick test when converting a working blade file to pug file does not work so far.

```html
<html>

<head>
    <livewire:styles />
</head>

<body>
    <livewire:counter />

    <livewire:scripts />
</body>

</html>
```

```pug
html

    head
        livewire:styles

    body
        livewire:counter

        livewire:scripts
```

Basically those livewire directives do not load further corresponding code.

**Comment by kylekatarnls** [comment]
Hello,

Note that `livewire:styles` is not HTML-standard so if you don't specify it, it will create a tag pair `<livewire:styles></livewire:styles>` to autoclose it you need to add a `/`

```pug
html

    head
        livewire:styles/

    body
        livewire:counter/

        livewire:scripts/
```

On default config, livewire tags are processed in blade templates so if you name your file with `.blade.pug` or register the `PugBladeCompiler` template engine for `.pug` you'll get anything rendered as in a blade template.

**Comment by anurat** [comment]
Thank you for your comment.

I tried adding a `/`, changed files from `*.pug` to `*.blade.pug`
but it still just generates a tag pair.

How do I register the PubBladeCompiler template engine for .pug?

**Comment by kylekatarnls** [comment]
It should now be fixed with the version 2.4.0.

**Comment by anurat** [comment]
Thank you so much!
Tried it out today and it seems to work with no problem so far!
